insights/parsers/cpupower_frequency_info.py

```diff
@@ -0,0 +1,88 @@
+"""
+CpupowerFrequencyInfo - Commands ``cpupower frequency-info``
```

Comment:
I believe the command should be `cpupower -c all frequency-info` to get the `frequency-info` of all the cpulist, Otherwise it only returns the info of CPU0.

Reply:
it's aligned to sosreport.